package.json

```diff
@@ -46,7 +46,7 @@
     "copy-webpack-plugin": "4.0.1",
     "coveralls": "^2.11.15",
     "css-loader": "^0.26.0",
-    "d3": "^4.4.1",
+    "d3": "^4.9.1",
```

Comment:
So, we pretty much *have* to put this in a separate bundle, ok.

Which means we'll need a webpack config change here to **not** include topology, in the dist bundle, and make a separate topology-only bundle...

.. and if we're doing that, maybe it's time to separate each component into its own bundle (`dialog-editor`, `gtl`, `site-switcher`, `toolbar` and `topology`, I guess), abandon the everything-bundle and include just the right ones in miq. (Then you'll need to put d3-4 and topology in a separate webpacker pack on the ui-side, and that's it.)

Cc @karelhala ^